### DISCUSSION TITLE
figure position in the right-top

### DISCUSSION THREAD

**Discussion by kongdd** [marp-team/marp#386] [Q&A] [answered]
I want to insert a small picture in the right-top (the red rect in the following figure) of my slice.
Anyone can teach me how to make it?

![image](https://user-images.githubusercontent.com/9815742/206626273-c88fdbd7-0d63-4a63-9522-dd7dcbc32bcb.png)

**Comment by yhatt** [comment, marked as answer]
There are several ways. If you want a one-shot placing of the image, defining simple image style in your slide like following, to put an image with [absolute positioning.](https://developer.mozilla.org/en-US/docs/Web/CSS/position#types_of_positioning)

```markdown
<style>
img[alt~="top-right"] {
  position: absolute;
  top: 30px;
  right: 30px;
}
</style>

# Your slide

![top-right](https://marp.app/assets/marp.svg)
```

`top` and `right` CSS declarations are meaning position of the image from edge. You can tweak `30px` if necessary.

> **Warning** This example is abusing `alt` attribute of `<img>` element, for making easy to target style to the image via Markdown syntax `![top-right]()`. If your slide output must consider users who using screen reader, recommend to avoid element narrowing by abused `alt` attribute selector. Instead, you can narrow the target based on the index number of images within the slide, powered by [scoped style](https://marpit.marp.app/theme-css?id=scoped-style) and [`:nth-of-type(n)`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type) CSS selector: `<style scoped>img:nth-of-type(1) { ... }</style>`

If you want to put the same image across multiple pages, just like a header image, https://github.com/marp-team/marp/discussions/200#discussioncomment-1535581 might be helpful.

**Comment by kongdd** [comment]
Awesome. Thank you very much. 
Marp saves me lots of time in making PPT. All of my course are maded by marp now, https://github.com/CUG-hydro/class2022_CUG_HydroMet
